engine/lib/comments.php

```diff
@@ -85,6 +89,67 @@ function _elgg_comments_page_handler($page) {
 }
 
 /**
+ * Redirect to the comment in context of the containing page
+ *
+ * @param int $comment_guid  GUID of the comment
+ * @param int $fallback_guid GUID of the containing entity
+ *
+ * @return void
+ * @access private
+ */
+function _elgg_comment_redirect($comment_guid, $fallback_guid) {
```

Comment:
These added functions are so similar but differ in so many ways I see no reason to put energy into refactoring.